kratos/python/add_matrix_to_python.cpp

```diff
@@ -89,6 +90,25 @@ namespace Python
 
           return matrix;
         }));
+
+        matrix_binder.def(py::init([](const py::list& input){
+          std::size_t num_rows = input.size();
+          KRATOS_ERROR_IF( num_rows == 0 ) << "Argument must be a Python's list of lists\n";
+          std::size_t num_cols = (Vector(py::cast<Vector>(input[0]))).size();
+
+          DenseMatrix<double>matrix = DenseMatrix<double>(num_rows, num_cols);
+          for(std::size_t i=0; i<num_rows; i++){
+            auto v_row = Vector(py::cast<Vector>(input[i]));
+            KRATOS_ERROR_IF( v_row.size() == 0 ) << "Empty Python-list of lists\n";
```

Comment:
So, if input is like `C = KM.Matrix( [[ ]] )`
will get `RuntimeError: Error: Empty Python-list of lists`

Reply:
The should give a matrix of size (0,0)
what do you think @rubenzorrilla ?

Reply:
Agree. Indeed, I think we should do the same for  `C = KM.Matrix([])`. In both cases what I'd expect is the same as if you do `C=KM.Matrix()`.

Reply:
Okay I will do that, In both the cases it will return Kratos.Matrix of size (0,0)